lib/chef/knife/ec2_server_create.rb

```diff
@@ -797,7 +794,11 @@ def bootstrap_common_params(bootstrap)
         bootstrap.config[:bootstrap_vault_item] = locate_config_value(:bootstrap_vault_item)
         bootstrap.config[:use_sudo_password] = locate_config_value(:use_sudo_password)
         bootstrap.config[:yes] = locate_config_value(:yes)
-        bootstrap.config[:tags] = config[:chef_tag] if locate_config_value(:chef_tag)
+        if locate_config_value(:chef_tag)
+          bootstrap.config[:tags] = config[:chef_tag]
```

Comment:
why we are using config[:chef_tag] and not locate_config_value[:chef_tag]

Reply:
Yes, we can use that. I will update the code